docs/misc/changelog.rst

```diff
@@ -9,6 +9,7 @@ Pre-Release 0.10.0a0 (WIP)
 
 Breaking Changes:
 ^^^^^^^^^^^^^^^^^
+- Renamed ``common.cmd_util`` to ``common.env_util`` for clarity
```

Comment:
missing an example of what is affected? (make_vec_env, make_atari_env) ?
+ maybe an explicit warning